### DISCUSSION TITLE
2D scopes

### DISCUSSION THREAD

**Discussion by em0rt41** [OGSR/OGSR-Engine#366] [Q&A]
Is there a way to get 2D scopes back? I feel that some pip scopes are really sticking like a sore thumb, not to mention the performance is not very friendly. Is there a way? Thanks.

**Comment by Kurzdor** [comment]
Try ```g_3d_scopes off``` in game console.

**Comment by em0rt41** [comment]
I've tried, but ACOG scope renders black. Some other scopes might be also rendering black, but I didn't try. I think the last update on the engine, 12 days ago, mentioned an update where this command you've mentioned is possible.

Pardon me and educate if needed, but I'm using [this](https://www.moddb.com/mods/ogsr-mod-repack-from-serega-lus-with-english-patch), it might be deprecated already, but I wouldn't know.

**Reply by xrSimpAI** [reply]
In OGSR Mod, grids for 2d scopes are not available everywhere (perhaps they are not at all)

**Comment by xrSimpAI** [comment]
> Is there a way to get 2D scopes back? I feel that some pip scopes are really sticking like a sore thumb, not to mention the performance is not very friendly. Is there a way? Thanks.

You can install the current version of the mod and configure the fps for 3D scopes in the graphics settings menu. This can significantly improve performance. Links to the current version are in our discord server.
